tests/storage.tests.js

```diff
@@ -12,93 +10,13 @@ describe('#write logs to storage', () => {
 
     store(storage, '').then(() => done());
   });
-});
 
-describe('#write logs to storage', () => {
-  it('should remove oldest logs and push news deploy to history', (done) => {
+  it('should update logs with 11 or more records', (done) => {
     const storage = {
-      read: () => Promise.resolve({ deployments: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 ] }),
-      write: (newData) => {
-        expect(newData.deployments.length).toBe(10);
-        expect(newData.deployments[0]).toBe(8);
-        return Promise.resolve({});
-      }
+      read: () => Promise.resolve({ deployments: [ 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 ] }),
+      write: () => Promise.resolve({})
     };
 
     store(storage, '').then(() => done());
   });
-
-  it('should remove oldest history when total history size exceeds 490KB, but not exceed length', (done) => {
-    var deployments = [];
```

Comment:
Why remove this test?

Reply:
I am yet to rewrite tests, i think i just started and this must of got caught up in a commit.